packages/d3fc-webgl/package.json

```diff
@@ -18,7 +18,8 @@
   },
   "dependencies": {
     "@d3fc/d3fc-rebind": "^5.0.8",
-    "d3": "^4.1.1"
+    "d3": "^4.1.1",
+    "glsl-transpiler": "^1.8.5"
```

Comment:
As this is only required for testing, please can you make it a devDependency?

Reply:
Sure.